platform/android/display_server_android.cpp

```diff
@@ -138,6 +138,11 @@ Size2i DisplayServerAndroid::screen_get_size(int p_screen) const {
 	return OS_Android::get_singleton()->get_display_size();
 }
 
+float DisplayServerAndroid::screen_get_refresh_rate(int p_screen) const {
+	// Most Android devices have 60hz displays.
```

Comment:
Quick search on gsmarena.com and, half of the last year released €200+ devices have 90+ Hz screens.

https://developer.android.com/reference/android/view/Display#getMode()
https://developer.android.com/reference/android/view/Display.Mode#getRefreshRate()